### PR TITLE
doc: add readme file for examples

### PR DESCRIPTION
**Pull request by jsbaguette**
### 🔗 Linked issue

https://github.com/nuxt/framework/issues/694
https://github.com/nuxt/framework/issues/715

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Not sure what's needed here. But a simple example what all readme can looks like as a template ? If we validate it I will also push readme for other examples

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by pi0** [comment]
Thanks for initial PR @f3ltron . Main purpose of this README files is to explain each example as an individual nuxt project. This way we can show them in an "Examples" tab of documentation. So basically for Hello world, we can explain you can play around with `App.vue` and `with-pages/` explain about pages and link to docs

**Comment by f3ltron** [comment]
ok so thats what I was thinking about. Is the format here is good enough for you, If you have a template in mind please give me ? I can continue to work on other examples project.

File: examples/hello-world/Readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## What you will find
+
+- A basic nuxt project that doesn't need pages.
+- When you don't need pages you can only use App.vue file.
+
+## Run the example
+
+from root folder:
+- `yarn`
```
Comment (pi0) [inline examples/hello-world/Readme.md:11]: It has to be
- `yarn install`
- `yarn dev`

(usage is assuming we are already inside `examples/hello-world`)

**Comment by pi0** [comment]
Thanks for the PR @jsbaguette (and SO sorry it got stalled this far). I'm closing this PR. We will rework this with #1960 and new Nuxt UI so that instructions can be both visible in playground and rendered documentation.

**Pull request closed without merging**